src/Storages/MergeTree/MergeTreeIndexInverted.cpp

```diff
@@ -626,6 +648,42 @@ bool MergeTreeConditionInverted::traverseASTEquals(
         out.set_gin_filters = std::move(gin_filters);
         return true;
     }
+    else if (function_name == "match")
```

Comment:
Does this code run if `allow_experimental_inverted_index` is not enabled? Let's make sure it is not run, and the same for `multiMatchAny`.